test/e2e/client.go

```diff
@@ -0,0 +1,74 @@
+package e2eutil
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"k8s.io/client-go/deprecated/scheme"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+
+	mdbv1 "github.com/mongodb/mongodb-kubernetes-operator/pkg/apis"
+)
+
+var TestClient client.Client
+var OperatorNamespace string
+var WatchNamespace string
+
+type CleanupOptions struct {
+	TestContext *Context
+}
+
+func (*CleanupOptions) ApplyToCreate(*client.CreateOptions) {}
```

Comment:
[q] what is this for?

Reply:
Ah this is a leftover from when I was passing a `CleanupOptions` object to `client.Create(...)` calls. I'll remove it.

Reply:
Resolving since this is necessary to implement the cleanup logic, even though we don't use it. CleanupOptions needs this method so that it matches the interface type passed to client.Create()